qtaf-core/pom.xml

```diff
@@ -234,7 +234,14 @@
         <dependency>
             <groupId>de.qytera</groupId>
             <artifactId>qtaf-io</artifactId>
-            <version>0.0.9</version>
+            <version>LOCAL_2023_04_27_001</version>
+            <scope>compile</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>de.qytera</groupId>
+            <artifactId>qtaf-security</artifactId>
+            <version>LOCAL_2023_04_27_001</version>
```

Comment:
Please don't push `pom` files containing development versions.

Reply:
Updated version to 0.1.0